tsconfig.json

```diff
@@ -1,21 +1,3 @@
 {
-	"compilerOptions": {
-		"module": "ESNext",
-		"lib": ["ESNext"],
-		"skipLibCheck": true,
-		"allowImportingTsExtensions": true,
-		"allowSyntheticDefaultImports": true,
-		"isolatedModules": true,
-		"moduleDetection": "force",
-		"noEmit": true,
-		"baseUrl": ".",
-		"strict": true,
-		"useDefineForClassFields": true
-	},
-
-	/* Linting */
-	"noUnusedLocals": true,
-	"noImplicitAny": true,
-	"noUncheckedIndexedAccess": true,
-	"noFallthroughCasesInSwitch": true
+	"extends": "../tsconfig.base.json"
```

Comment:
I guess path should point to local dir `./tsconfig.base.json`

Reply:
oops, yes, thanks.